src/__fixtures__/caip-types.ts

```diff
@@ -33,3 +33,37 @@ export const CAIP_ACCOUNT_ID_FIXTURES = [
 export const CAIP_ACCOUNT_ADDRESS_FIXTURES = Array.from(
   new Set(CAIP_ACCOUNT_ID_FIXTURES.map((value) => value.split(':')[2])),
 );
+
+export const CAIP_ASSET_ID_FIXTURES = [
```

Comment:
This list currently combines both `asset types` and `asset IDs`. It might be clearer to rename it to `CAIP_ASSET_TYPE_AND_ID_FIXTURES`. However, it would likely be more useful to maintain two separate lists, one for types and another for IDs, so you can use them independently in the tests below.

Reply:
Note: it was fixed with commit `225dff4`